osquery/events/events.cpp

```diff
@@ -566,6 +566,10 @@ Status EventSubscriberPlugin::add(Row& r, EventTime event_time) {
 
   r["time"] = std::to_string(event_time);
   r["eid"] = eid;
+  // The field below is added for event forwarding to tls logger.
+  // Since this field is not part of any event table definition,
+  // so this does not exist for upper schedule query layer.
+  r["event_type"] = getName();
```

Comment:
Since this is not a valid column in the spec, use something to differentiate. Maybe `_event_type`.